main.py

```diff
@@ -111,26 +111,21 @@ def produce_kafka_messages(bucket, job_id, fixture_data, args):
     )
     s3_client = boto3.client("s3")
     for s3_key in fixture_data:
-        logger.debug(f"Processing key: {s3_key}")
-        line_no = 0
-        for line in (
-            s3_client.get_object(Bucket=bucket, Key=s3_key)["Body"]
-            .read()
-            .splitlines()
-        ):
-            line_no += 1
-            try:
-                data = json.loads(line)
-                db = data["message"]["db"] if "db" in data["message"] else "missingDb"
-                collection = data["message"]["collection"] if "collection" in data["message"] else "missingCollection"
-                topic_name = f"{args.topic_prefix}{job_id}_{db}.{collection}"
-            except json.JSONDecodeError as err:
-                logger.error(
-                    f"line {line_no} of {s3_key} contains invalid JSON data: {err.msg}"
-                )
-                continue
-            producer.send(topic_name, line)
-            producer.flush()
+        logger.info(f"Processing key: {s3_key}")
+        file = s3_client.get_object(Bucket=bucket, Key=s3_key)["Body"].read()
+        try:
+            data = json.loads(file)
+            db = data["message"]["db"] if "db" in data["message"] else "missingDb"
+            collection = data["message"]["collection"] if "collection" in data["message"] else "missingCollection"
+            topic_name = f"{args.topic_prefix}{job_id}_{db}.{collection}"
+        except json.JSONDecodeError as err:
+            logger.error(
+                f"line {s3_key} contains invalid JSON data: {err.msg}"
```

Comment:
Generecize this now that we're not doing JSON-parsing.

Reply:
we do need to parse the file incase it is son so that we can *try* and get the db and collection names out, if possible.